Makefile

```diff
@@ -32,7 +32,7 @@ CHECKSUM=$(shell md5sum $(SRC_DEPS) | md5sum | awk '{print $$1}')
 PKG:=k8s.io/client-go/pkg
 LDFLAGS:=-X $(PKG)/version.gitVersion=$(GIT_TAG) -X $(PKG)/version.gitCommit=$(GIT_COMMIT) -X $(PKG)/version.buildDate=$(BUILD_DATE)
 
-metrics-server: $(SRC_DEPS)
+metrics-server:
```

Comment:
Let's keep this rule for caching to work

Reply:
Intead let's remove `go.mod` from `pkg/api/generated/openapi/zz_generated.openapi.go: go.mod` line

Reply:
sounds good